app/uk/gov/hmrc/merchandiseinbaggage/views/components/inputSelect.scala.html

```diff
@@ -37,7 +37,9 @@
 @govukSelect(Select(
     id = id,
     name = name,
-    items = Seq(SelectItem.defaultObject) ++ items,
+    items = Seq(
```

Comment:
this label would show in english when on a welsh toggle, is there welsh for this message?

Reply:
![Screenshot 2023-11-01 at 13 37 01](https://github.com/hmrc/merchandise-in-baggage-frontend/assets/47248104/67d4c212-f7b5-4fe2-b78f-b76283c32cd4)

Reply:
also think the message should be called in from the messages file rather than hard-coded in the component

Reply:
![Screenshot 2023-11-01 at 13 44 54](https://github.com/hmrc/merchandise-in-baggage-frontend/assets/47248104/02c4b304-69e5-4d8b-8851-c8a12b15afc6)

similar issue with the country look up because it uses the same component